index.bs

```diff
@@ -85,7 +85,7 @@ THE OTHER MEMBER HAS BEEN ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 
 <pre class="anchors">
 url: #biblio-heif; spec: HEIF; type: property;
-    text: colr
+    text: colr 
```

Comment:
A space character seems to have been added accidentally.

Reply:
fixed